<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ploading</class>
 <widget class="QWidget" name="ploading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Distribution Profile:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="itgaus">
      <item>
       <property name="text">
        <string>Gaussian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uniform</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Parabolic</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Energy Profile:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="itgamgaus">
      <item>
       <property name="text">
        <string>Gaussian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uniform</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>IALL:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="iall"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>IPSEED:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>NBINS:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="ipspeed"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="nbins"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>particle phase</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>energy distribution</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>x</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>y</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>px</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>py</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Hammersley Base</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
